src/translate.rs

```diff
@@ -1162,38 +1180,77 @@ impl FromGlib<i32> for Option<u64> {
     }
 }
 
-/// Translate from a pointer type without taking ownership, transfer: none.
+/// Translate from a pointer type which is annotated with `transfer none`.
+/// The resulting value is referenced at least once, by the bindings.
+///
+/// This is suitable for floating references, which become strong references.
+/// It is also suitable for acquiring non-gobject values, like `gchar*`.
+///
+/// The implementation of this trait should acquire a reference to the value
+/// in a way appropriate to the type, e.g. by referencing or copying.
+/// Values obtained using this trait must be properly unreferenced on `drop()`
```

Comment:
"unreferenced" here, "released" in "full",  and again "unreferenced" in "borrow"

Reply:
Thank you, fixed.